compiler/src/dotty/tools/dotc/transform/GenericSignatures.scala

```diff
@@ -127,6 +127,9 @@ object GenericSignatures {
         case ref @ TypeParamRef(_: PolyType, _) =>
           typeParamSig(ref.paramName.lastPart)
 
+        case RefOrAppliedType(sym, _, _) if defn.isXXLFunctionClass(sym) =>
```

Comment:
Shouldn't you add it as a new case in the already existing `if else` chain starting line 86

Reply:
Indeed